### PR TITLE
Incorporate packet structure in sensors module

### PR DESCRIPTION
**Pull request by stephen99scott**
* Review after #93 and #95

### CODE REVIEW

**Comment by adambujak** [comment]
can you please merge the other PRs and then rebase - it's hard to see what's new here

**Comment by stephen99scott** [comment]
@adambujak Should be good now

File: firmware/applications/SignBuddy/sensors.c [hunk 1]
```diff
@@ -22,12 +23,14 @@
...
 
-#define SAMPLING_PERIOD      1000
+#define SAMPLING_PERIOD      50
+#define MAX_SAMPLES          40
 
 typedef struct {
   TaskHandle_t       sensors_task_handle;
   EventGroupHandle_t data_ready_event_group;
   TimerHandle_t      sampling_timer;
+  Sample             sample;
```
Comment (adambujak) [inline firmware/applications/SignBuddy/sensors.c:33]: this type name is not great - is it defined in the proto?

**Review by adambujak — APPROVED** [review]

File: firmware/applications/SignBuddy/sensors.c (hunk 1 above)
Comment (stephen99scott) [reply firmware/applications/SignBuddy/sensors.c:33]: Yeah it's defined in the proto

**Pull request merged**